airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -62,6 +62,23 @@
     from airflow.utils.context import Context
 
 
+def _task_id_to_pod_name(val):
```

Comment:
this is from https://github.com/apache/airflow/pull/27524 which should be merged first